src/audits/performance/user-timings.js

```diff
@@ -41,43 +38,18 @@ class UserTimings extends Audit {
    * @return {!AuditResult}
    */
   static audit(artifacts) {
-    if (!artifacts.traceContents || !Array.isArray(artifacts.traceContents)) {
+    if (typeof artifacts.userTimings === 'undefined' ||
+      !Array.isArray(artifacts.userTimings)) {
       return UserTimings.generateAuditResult({
```

Comment:
if you add some kind of error message from the user timings gather, should pipe through to `debugString` here